diff --git a/memcache_admin/src/Controller/MemcacheStatisticsController.php b/memcache_admin/src/Controller/MemcacheStatisticsController.php
index 7e119da5f836f1c4c04e98a6a11f1433ac976c89..54a6780fb982d7afd09cc0593927b79ac92ee3c6 100644
--- a/memcache_admin/src/Controller/MemcacheStatisticsController.php
+++ b/memcache_admin/src/Controller/MemcacheStatisticsController.php
@@ -375,8 +375,8 @@ class MemcacheStatisticsController extends ControllerBase {
       [
         '@get'   => number_format($get, 2),
         '@set'   => number_format($set, 2),
-        '@read'  => format_size(number_format($read, 2)),
-        '@write' => format_size(number_format($write, 2)),
+        '@read'  => format_size((int) number_format($read, 2)),
+        '@write' => format_size((int) number_format($write, 2)),
       ]
     );
   }
diff --git a/src/Driver/MemcachedDriver.php b/src/Driver/MemcachedDriver.php
index f6a9a699c8c8abd3d5244ecf7e19a55f4d4d1dcb..07df5bf012bb7cddb974210ac91d1a5800305419 100755
--- a/src/Driver/MemcachedDriver.php
+++ b/src/Driver/MemcachedDriver.php
@@ -68,7 +68,7 @@ class MemcachedDriver extends DriverBase {
       }
     }

-    if (PHP_MAJOR_VERSION === 7) {
+    if (PHP_MAJOR_VERSION >= 7) {
       $results = $this->memcache->getMulti($full_keys, \Memcached::GET_PRESERVE_ORDER);
     }
     else {
